lib/openxml/docx/package.rb

```diff
@@ -22,6 +22,10 @@ def initialize
         @settings = OpenXml::Docx::Parts::Settings.new
         @styles = OpenXml::Docx::Parts::Styles.new
         @document = OpenXml::Docx::Parts::Document.new
+        @styles << default_table_style
```

Comment:
Should this _always_ be in Styles?

If so, let's move it into `OpenXml::Docx::Parts::Styles`

If it's not strictly needed, then maybe we should pull it out to `examples/table` (i.e. leave it up to the consumer?)